src/include/storage/garbage_collector.h

```diff
@@ -93,7 +105,7 @@ class GarbageCollector {
   // queue of txns that need to be unlinked
   transaction::TransactionQueue txns_to_unlink_;
   // queue of unexecuted deferred actions
-  std::queue<std::pair<transaction::timestamp_t, transaction::Action>> deferred_actions_;
+  std::queue<std::pair<transaction::timestamp_t, transaction::TransactionAction>> deferred_actions_;
```

Comment:
See earlier note about changing label to “PlannedAction”. It seems like you’re using TransactionAction in a conflicting way.

Reply:
This is unused. Seems like a merge problem. Removed.